src/content/docs/bff/fundamentals/session/handlers.mdx

```diff
@@ -11,6 +11,58 @@ redirect_from:
   - /identityserver/v6/bff/session/handlers/
   - /identityserver/v7/bff/session/handlers/
 ---
+import { Badge } from "@astrojs/starlight/components";
+import { Code } from "@astrojs/starlight/components";
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+To configure authentication in the BFF, you'll need to configure both the OpenID Connect 
+login flow and the cookie handlers. 
+
+## Automatic authentication configuration <Badge text="V4" />
+
+In V4, a simplified mechanism for wiring up authentication has been introduced. The main purpose for the BFF
+is to handle the OpenID Connect login flow and to protect the api's using Cookies. In V3, you explicitly had 
+to configure the aspnet core authentication system to enable this, but in V4, this is now simplified.
+
+A call `BffBuilder.ConfigureOpenIdConnect()` will make sure that:
+1. The authentication pipeline is configured with the appropriate authentication schemes. 
+2. The OpenID Connect pipeline is configured with default values. 
+3. The CookieHandler is configured using recommended practices. This can be tweaked by calling `BffBuilder.ConfigureCookies()`
+
+Below is an example on how to configure the BFF's authentication pipeline. 
+
+```csharp
+
+services.AddBff()
+    .ConfigureOpenIdConnect(options =>
+    {
+        options.Authority = "https://demo.duendesoftware.com";
+        options.ClientId = "interactive.confidential";
+        options.ClientSecret = "secret";
+        options.ResponseType = "code";
+        options.ResponseMode = "query";
+
+        options.GetClaimsFromUserInfoEndpoint = true;
+        options.SaveTokens = true;
+        options.MapInboundClaims = false;
+
+        options.Scope.Clear();
+        options.Scope.Add("openid");
+        options.Scope.Add("profile");
+        options.Scope.Add("api");
+        options.Scope.Add("offline_access");
+
+        options.TokenValidationParameters.NameClaimType = "name";
+        options.TokenValidationParameters.RoleClaimType = "role";
+    });
+```
+
+Each frontend can have custom OpenID Connect configuration and Cookie Configuration. This can both be configured programmatically 
+as via [Configuration](../multi-frontend/configuration.mdx). 
```

Comment:
```suggestion
Each frontend can have custom OpenID Connect configuration and Cookie Configuration. This can both be configured programmatically 
via [Configuration](../multi-frontend/configuration.mdx). 
```